tests/rptest/services/redpanda.py

```diff
@@ -1666,6 +1666,21 @@ def scale_cluster(self, nodes_count):
         """
         return self._cloud_cluster.scale_cluster(nodes_count)
 
+    def clean_cluster(self):
+        """Cleans state from a running cluster to make it seem like it was newly provisioned.
+
+        Call this function at the end of a test case so the next test case has a clean cluster.
+        Be careful calling this method if you expect another test to be running and modifying the same cluster concurrently.
+        """
+
+        # assuming topics beginning with '_' are system topics that should not be deleted
+        rpk = RpkTool(self)
+        topics = rpk.list_topics()
+        deletable = [x for x in topics if not x.startswith('_')]
+        self.logger.debug(f'found {len(deletable)} topics to delete')
```

Comment:
I think it's worth listing them.

Reply:
I would even list some high level stats, but that's an overkill.

Reply:
Is there a way to do `rpk cluster topic delete -r ".*"` in our wrapper which does the right thing?